DataFormats/Phase2TrackerRecHit/interface/Phase2TrackerRecHit1D.h

```diff
@@ -0,0 +1,33 @@
+#ifndef DATAFORMATS_PHASE2TRACKERRECHIT_PHASE2TRACKERRECHIT1D_H 
+#define DATAFORMATS_PHASE2TRACKERRECHIT_PHASE2TRACKERRECHIT1D_H 
+
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+
+#include "DataFormats/Phase2TrackerCluster/interface/Phase2TrackerCluster1D.h"
+#include "DataFormats/GeometrySurface/interface/LocalError.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+
+typedef edm::Ref< edmNew::DetSetVector< Phase2TrackerCluster1D >, Phase2TrackerCluster1D > Phase2ClusterReference;
+
+class Phase2TrackerRecHit1D {
```

Comment:
no connection to TrackingRecHit?
Why not? was there an issue with persistence or something else?